zipkin-server/src/main/java/zipkin2/server/internal/health/ZipkinHealthController.java

```diff
@@ -95,19 +91,15 @@ static HttpResponse newHealthResponse(List<ComponentHealth> healths, MediaType m
     return newHealthResponse(overallStatus, mediaType, healthJson);
   }
 
-  static HttpResponse newHealthResponse(String overallStatus, MediaType mediaType,
-    String healthJson) {
-    byte[] body = healthJson.getBytes(UTF_8);
-    int code = overallStatus.equals(STATUS_UP) ? 200 : 503;
-    ResponseHeaders headers = ResponseHeaders.builder(code)
-      .contentType(mediaType)
-      .setInt(CONTENT_LENGTH, body.length).build();
-    return HttpResponse.of(AggregatedHttpResponse.of(headers, HttpData.wrap(body)));
+  static HttpResponse newHealthResponse(String status, MediaType mediaType, String healthJson) {
+    HttpStatus code = status.equals(STATUS_UP) ? HttpStatus.OK : HttpStatus.SERVICE_UNAVAILABLE;
+    return HttpResponse.of(code, mediaType, healthJson);
   }
 
   static String writeJsonError(String error) throws IOException {
     StringWriter writer = new StringWriter();
     JsonGenerator generator = JSON_FACTORY.createGenerator(writer);
+    generator.useDefaultPrettyPrinter();
```

Comment:
I believe calling this every time is equivalent to this on the factory

`.enable(SerializationFeature.INDENT_OUTPUT);`

Reply:
actually that is in the databind package.